File: engine/engine-model/src/main/scala/tools/packet/message/ReceiveFame.scala
package tools.packet.message

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class ReceiveFame(private var _mode: Int, private var _characterNameFrom: String) extends PacketInput {
  def mode: Int = _mode

  def characterNameFrom: String = _characterNameFrom

  override def opcode(): SendOpcode = SendOpcode.FAME_RESPONSE
}

File: engine/engine-model/src/main/java/tools/packet/message/ReceiveFame.java
package tools.packet.message;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record ReceiveFame(Integer mode, String characterNameFrom) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.FAME_RESPONSE;
   }
}